GoogleDataTransportCCTSupport/GDTCCTLibrary/Protogen/nanopb/cct.nanopb.c

```diff
@@ -30,10 +30,11 @@ const gdt_cct_QosTierConfiguration_QosTier gdt_cct_LogRequest_qos_tier_default =
 const int32_t gdt_cct_QosTierConfiguration_log_source_default = 0;
 
 
-const pb_field_t gdt_cct_LogEvent_fields[6] = {
+const pb_field_t gdt_cct_LogEvent_fields[7] = {
```

Comment:
I suppose this is generated code, but it seems weird ...

Aren't there 8 elements including PB_LAST_FIELD?  And why not `gdt_cct_LogEvent_fields[]` to let the compiler set the length?

Reply:
I only count 7...

```
1    PB_FIELD(  1, INT64   , OPTIONAL, STATIC  , FIRST, gdt_cct_LogEvent, event_time_ms, event_time_ms, 0),
2    PB_FIELD(  6, BYTES   , OPTIONAL, POINTER , OTHER, gdt_cct_LogEvent, source_extension, event_time_ms, 0),
3    PB_FIELD( 11, INT32   , OPTIONAL, STATIC  , OTHER, gdt_cct_LogEvent, event_code, source_extension, 0),
4    PB_FIELD( 15, SINT64  , OPTIONAL, STATIC  , OTHER, gdt_cct_LogEvent, timezone_offset_seconds, event_code, 0),
5    PB_FIELD( 17, INT64   , OPTIONAL, STATIC  , OTHER, gdt_cct_LogEvent, event_uptime_ms, timezone_offset_seconds, 0),
6    PB_FIELD( 23, MESSAGE , OPTIONAL, STATIC  , OTHER, gdt_cct_LogEvent, network_connection_info, event_uptime_ms, &gdt_cct_NetworkConnectionInfo_fields),
7    PB_LAST_FIELD
```

it's weird that it diff'd as a removal and addition of a line, when it should just be an addition, though.

Reply:
I'd also add that nanopb was written for embedded systems, some of which might have not-up-to-spec compilers.